src/it/scala/io/iohk/ethereum/ledger/BlockImporterItSpec.scala

```diff
@@ -153,6 +168,34 @@ class BlockImporterItSpec
     blockchain.getBestBlock().get shouldEqual newBlock3
   }
 
+  it should "return Unknown branch, in case of PickedBlocks with block that has a parent that's not in the chain" in {
+    val newcomerBlock4: Block =
+      getBlock(genesisBlock.number + 4, difficulty = 104, parent = oldBlock3.header.hash)
+    val newcomerWeight4Duplicate = oldWeight3.increase(newcomerBlock4.header)
+
+    //Block n5 with oldBlock4 as parent
+    val newComerBlock5WithOldBlock4Parent: Block =
```

Comment:
maybe it would make sense to propose a shorthand for those and stick to it? eg.
`newBlock5ParentOldBlock4`?
`newcomerBlock4` would be `newBlock4ParentOldBlock3`

just a thought